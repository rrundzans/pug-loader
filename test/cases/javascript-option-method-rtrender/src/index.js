// require with alias 'Template'
const html = require('Template/widget.pug?{"a":10,"b":"abc"}');

console.log(html);