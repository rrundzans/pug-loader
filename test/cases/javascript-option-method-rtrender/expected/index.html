<h1>Hello world!</h1><span>Param[a]: 10</span><span>Param[b]: abc</span><img src="assets/images/image-f78b30f412cd192e6753.jpeg">